source/target/realtek/rtl8195am/flash_blob.c

```diff
@@ -0,0 +1,79 @@
+    0xf898d1f6, 0x21000000, 0xf0106031, 0xd0bc0f02, 0x0102e9dd, 0x9a012300, 0xf8df6033, 0x2301c318,
+    0x3000f8cc, 0xf423683b, 0xf4237340, 0x603b2370, 0x2000f888, 0x28002200, 0x5c8bd906, 0x3000f888,
+    0xb2d21c52, 0xd8f84290, 0x60302001, 0xf8cd2000, 0x6821a000, 0xf0011c40, 0x91000101, 0xd80142a8,
+    0xd1f62900, 0xf931f000, 0x60302000, 0xf8cb2001, 0x20050000, 0x60312100, 0xf4216839, 0xf4412170,
+    0x60397140, 0x0000f888, 0x60302001, 0xf8cd2000, 0x6821a000, 0xf0011c40, 0x91000101, 0xd80142a8,
+    0xd1f62900, 0x0000f898, 0x60312100, 0x0f02f010, 0xb004d1da, 0x8df0e8bd, 0xb0814a9f, 0x68112000,
+    0x29219100, 0x2121d004, 0x1c406011, 0xdbf62814, 0x6801489a, 0x0101f041, 0x49996001, 0xf043680b,
+    0x600b0310, 0x680b4997, 0x7380f443, 0x680b600b, 0x7300f443, 0x4b94600b, 0xf0416819, 0x60190120,
+    0xf0216801, 0x60010106, 0xf3c06810, 0x28031002, 0x2003bf28, 0x444a4a8d, 0x0c04f1a2, 0x1020f812,
+    0x0080eb02, 0x1000f88c, 0xf88c7842, 0x78802001, 0x0002f88c, 0x68004886, 0x0f01f010, 0x4885d016,
+    0xf0106800, 0xd0110f20, 0x68034883, 0x7f80f013, 0x6800d00c, 0x4007f3c0, 0x031ff000, 0x0343eb03,
+    0x3002f88c, 0x1050eb02, 0x0001f88c, 0x4a6e2301, 0x3003f88c, 0x60102000, 0x60014878, 0x60031f00,
+    0x68084977, 0xf89c9000, 0xf36f3001, 0x4318000f, 0x49746008, 0x90006808, 0x3002f89c, 0x00fff020,
+    0x60084318, 0x1f112004, 0x20006008, 0x68191f0b, 0x117cf421, 0x60199100, 0x4a6b6010, 0x60112138,
+    0x21eb4a6a, 0x4a6a6011, 0xf4216811, 0x601171ff, 0x4770b001, 0x47702000, 0x2200b510, 0x20c74611,
+    0xfe9ef7ff, 0xbd102000, 0x39080b01, 0xbf9c2902, 0x47702000, 0xf1a0b508, 0x0c0140d0, 0x1000f88d,
+    0xf88d0a01, 0xf88d1001, 0x466a0002, 0x20202103, 0xfe86f7ff, 0xbd082000, 0x4df7e92d, 0xb0812400,
+    0xd94e2900, 0xe14cf8df, 0xf8df4f41, 0x46a38100, 0x0a04f1ae, 0x0604f10e, 0x5530f247, 0x19039801,
+    0x43d0f1a3, 0x60186810, 0xf8cd2000, 0x683bb000, 0xf0031c40, 0x93000301, 0xd80142a8, 0xd1f62b00,
+    0x0c00f04f, 0x60302000, 0xf8ce2001, 0x20050000, 0x60332300, 0x3000f8da, 0x2370f423, 0x7340f443,
+    0x3000f8ca, 0x0000f888, 0x60302001, 0xf8cd2000, 0x683bb000, 0xf0031c40, 0x93000301, 0xd80142a8,
+    0xd1f62b00, 0x0000f898, 0x60332300, 0x0f01f010, 0xf10cbf1c, 0x45650c01, 0x1d12d8d4, 0x42a11d24,
+    0xb004d8bc, 0xe8bd2000, 0xe92d8df0, 0xf8df01f8, 0x220080a4, 0x4c154d16, 0xf1a84617, 0xf1080604,
+    0xf2470c04, 0x20005330, 0x0000f8cc, 0xf8c82001, 0x20050000, 0xf8cc2100, 0x68311000, 0x2170f421,
+    0x7140f441, 0x70206031, 0xf8cc2001, 0x20000000, 0x68299700, 0xf0011c40, 0x91000101, 0xd8014298,
+    0xd1f62900, 0xe0277820, 0x40006008, 0x40006060, 0x40006028, 0x40006118, 0x40000014, 0x400002c0,
+    0x40000210, 0x40000230, 0x40000320, 0x00000008, 0x400001f4, 0x400000e0, 0x40000038, 0x40006014,
+    0x4000611c, 0x40000300, 0x40006104, 0x400060f0, 0x40006120, 0x40006004, 0xf8cc2100, 0xf0101000,
+    0xd0060f01, 0x42931c52, 0xe8bdd8ad, 0x200001f8, 0xe8bd4770, 0x200101f8, 0x00004770, 0x00000000,
+    0x00000000, 0x005e0101, 0x00000101, 0x00230201, 0x00050501
```

Comment:
Can you provide details on where this flash algo was created from?

Reply:
From a template in flashalgo repository + RTL8195AM SPI-F control drivers.

Reply:
I don't see RTL8195AM SPI-F control drivers in the flash algo repository. Was this done locally?

Reply:
It's done locally. Code has been transferred to RTL8195AM team. They will decide when and whether to upstream.